homeassistant/components/lifx/light.py

```diff
@@ -448,9 +448,33 @@ def __init__(self, bulb, effects_conductor):
         """Initialize the light."""
         self.bulb = bulb
         self.effects_conductor = effects_conductor
-        self.registered = True
+        self._attr_available = True
```

Comment:
The `registered` property is actually accessed from outside this class.

Reply:
Done

Reply:
`self._attr_available = True` is the default, and should not be defined here.

Reply:
DOne